packages/golden_toolkit/.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
 	"editor.insertSpaces": false,
-	"editor.wordWrapColumn": 120
+	"editor.wordWrapColumn": 80,
+	"dart.lineLength": 80
```

Comment:
I know why we are doing this and I agree with the reasoning but man I hate how 80 char lines look.

Reply:
I know 😢